packages/cli/src/commands/upgrade.js

```diff
@@ -28,12 +33,12 @@ export const builder = (yargs) => {
       type: 'string',
       coerce: validateTag,
     })
-    .option('pr', {
-      description: 'Installs packages for the given PR',
-      requiresArg: true,
-      type: 'string',
+    .option('verbose', {
+      alias: 'vvv',
```

Comment:
```suggestion
      alias: 'v',
```

Reply:
-v goes to version I believe :)

Reply:
Nevermind, mixed it up with something else